File: Daily Challanges/2026-02-06.js
function getFlag(code) {
  code = code.toUpperCase();
  return [...code].map(c => String.fromCodePoint(c.charCodeAt(0) + 127397)).join('');
}

const test1 = getFlag("AL");
const test2 = getFlag("AD");
const test3 = getFlag("AR");
const test4 = getFlag("AM");
const test5 = getFlag("AU");

const check1 = "🇦🇱";
const check2 = "🇦🇩";
const check3 = "🇦🇷";
const check4 = "🇦🇲";
const check5 = "🇦🇺";

console.log(`Test1: ${test1 === check1}`);
console.log(`Test2: ${test2 === check2}`);
console.log(`Test3: ${test3 === check3}`);
console.log(`Test4: ${test4 === check4}`);
console.log(`Test5: ${test5 === check5}`);

File: Daily Challanges/2026-02-February/2026-02-06.py
def get_flag(code):
    code = code.upper()
    return ''.join(chr(ord(c) + 127397) for c in code)

test1 = get_flag("AL")
test2 = get_flag("AD")
test3 = get_flag("AR")
test4 = get_flag("AM")
test5 = get_flag("AU")

check1 = "🇦🇱"
check2 = "🇦🇩"
check3 = "🇦🇷"
check4 = "🇦🇲"
check5 = "🇦🇺"

print(f"Test1: {test1 == check1}")
print(f"Test2: {test2 == check2}")
print(f"Test3: {test3 == check3}")
print(f"Test4: {test4 == check4}")
print(f"Test5: {test5 == check5}")